ansible/roles/docker_storage_setup/tasks/main.yml

```diff
@@ -1,17 +1,18 @@
 ---
 - name: Ensure docker is installed
-  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
-  with_items:
-  - docker
-  #- docker-1.9.1
+  package:
+    name: docker
+    state: present
 
-- name: copy the docker-storage-setup config file
+- name: Copy the docker-storage-setup config file
   template:
-    src: docker-storage-setup.j2
+    src: "{{ item }}"
     dest: /etc/sysconfig/docker-storage-setup
     owner: root
     group: root
     mode: 0664
+  with_first_found:
+    - "{{ dss_docker_storage_driver }}_dss.j2"
```

Comment:
Is there a reason to use `with_first_found` here?

If we only have 1 in the list, why not just put this as the `src` to the module?

The `template` module will error out if it doesn't find this file as the `src`.

Reply:
Just a vestige of having a fallback file there.  I'll fix.

Good to merge after that?